PCampReview.py

```diff
@@ -158,15 +155,80 @@ def infoPopup(self,message):
     messageBox = qt.QMessageBox()
     messageBox.information(None, 'Slicer mpMRI review', message)
 
+  def setupIcons(self):
```

Comment:
I agree, is there any way we can shrink/tighten the tabs a little bit?  We've managed to save vertical space by using tabs instead of collapse boxes, but we've grown horizontally.